### PR TITLE
ci(zuul): fix reverse-dependency tests

### PR DESCRIPTION
**Pull request by mfocko**
Related to packit/packit#2008

### CODE REVIEW

File: files/install-deps.yaml [hunk 1]
```diff
@@ -40,6 +36,14 @@
         name:
           - sentry-sdk[flask]
           - syslog-rfc5424-formatter
+          - sqlalchemy
+          # postgresql driver that was included in the RPM package
+          - psycopg2
```
Comment (nforro) [inline files/install-deps.yaml:41]: Shouldn't `python3-sqlalchemy+postgresql` above be removed?

**Review by nforro — APPROVED** [review]

File: files/install-deps.yaml (hunk 1 above)
Comment (mfocko) [reply files/install-deps.yaml:41]: ah, must've missed it, thanks

Comment (nforro) [reply files/install-deps.yaml:40]: I think it's also possible to specify an extra to pip like this: `sqlalchemy[postgresql]`
But the result would be the same.

**Comment by mfocko** [comment]
:+1: to move the openshift tests to TF :smile:

File: files/install-deps.yaml (hunk 1 above)
Comment (jpopelka) [reply files/install-deps.yaml:40]: Won't don't need to install them with pip, https://github.com/packit/packit-service/pull/2061/commits/271f3622c8e540264313fcf2c609e10814c2ab50 should be sufficient I think.

Comment (mfocko) [reply files/install-deps.yaml:40]: I went through the SQLAlchemy docs just to find that they include the postgresql API by default in SQLAlchemy with the exception of the DB driver which is the psycopg2

The fact I wasn't able to tell the difference between `+something` RPM packages by looking at specfile just frustrated me even more :D

**Comment by jpopelka** [comment]
I'm trying to rebase #2061 which should solve the tests as well.

File: files/install-deps.yaml (hunk 1 above)
Comment (nforro) [reply files/install-deps.yaml:40]: > The fact I wasn't able to tell the difference between `+something` RPM packages by looking at specfile just frustrated me even more :D

That's the way how to specify extras (it's all automatic based on package metadata, there doesn't have to be anything in a spec except for a `%python_extras_subpkg` macro): https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#Extras

**Pull request closed without merging**